tests/utils/engine_process.py

```diff
@@ -32,6 +32,23 @@ def _check_models_api(self, response):
         except Exception:
             return False
 
+    def curl_metrics_endpoint(self, port=8081):
```

Comment:
```suggestion
    def get_metrics(self, port=8081):
```
To match up better with send_request